www/index.js

```diff
@@ -47,14 +47,21 @@ const canvas = document.createElement('canvas');
 canvas.id = 'canvas';
 document.body.appendChild(canvas);
 
-const game = wasm.make_game()
+const maybeGame = wasm.make_game();
+if (maybeGame.get_error()) {
```

Comment:
I get it but it is so verbose in JS 😳

Reply:
I think I've found a way to do it in a natural way in JS: https://rustwasm.github.io/wasm-bindgen/reference/types/result.html

If we return a Result<T, JsValue> then it will either return the T or throw the JsValue